.yamato/config/hdrp.metafile

```diff
@@ -4,14 +4,23 @@ project:
   folder_standalone: HDRP_RuntimeTests
 test_platforms:
   - type: playmode
+  - type: playmode
+    name: playmode_cache
+    utr_flags:
+    - [all]: --extra-editor-arg="-adb2"
+    - [all]: --extra-editor-arg="-enableCacheServer"
+    - [all]: --extra-editor-arg="-cacheServerEndpoint cacheserver-slo.hq.unity3d.com"
+    - [all]: --extra-editor-arg="-cacheServerNamespacePrefix SRP"
+    - [all]: --extra-editor-arg="-cacheServerEnableDownload true"
+    - [all]: --extra-editor-arg="-cacheServerEnableUpload true"
   - type: playmode
     name: playmode_XR
     utr_flags:
       - [all]: --extra-editor-arg="-xr-tests" 
   - type: editmode
   - type: Standalone
-    utr_flags:
-      - [OSX_Metal]: --timeout=2400
+    timeout:
```

Comment:
This might be a merge issue, we've removed the timeout I think, and its set as flag now?

Reply:
ah thank you!

Reply:
so the bit that was removed is what we want?

Reply:
Sorry if this is intentional but the new change introduced timeout differences on Android and Iphones: https://github.com/Unity-Technologies/Graphics/pull/2113/files#diff-e6cef0d182d76fce2e2f426c1b980215835d5affe424f8fd55b87aa2b154503cR63

Reply:
i will fix, thanks :)

Reply:
ah, i missed `- [all]: --timeout=3000`